### PR TITLE
[10.0] partner noncommercial

### PR DESCRIPTION
**Pull request by NL66278**
Migrated module from 8.0.

Module renames commercial names for Sales menu to Partners, and also limits Customers to people actually doing sales, while other users get to choose between Organisations, persons or all partners.

All in all a few simple changes that make the UI more appropiate for non commercial organisations.

### CODE REVIEW

**Review by gonzaloruzafa — APPROVED** [review]
+1 Functional Testing Ok.

File: partner_noncommercial/data/ir_translation.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo><data>
```
Comment (leemannd) [inline partner_noncommercial/data/ir_translation.xml]: <data> can be removed

File: partner_noncommercial/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Therp BV (http://therp.nl).
```
Comment (leemannd) [inline partner_noncommercial/__manifest__.py]: s/©/Copyright

**Review by leemannd — APPROVED** [review]
Thanks for the port. Only nitpicking.
Agreed once changed

**Comment by NL66278** [comment]
@leemannd Thanks for your review. Changes have been done.

**Comment by NL66278** [comment]
@Yajo If you agree requested changes have been done, could you remove Needs Fixing and Needes Review labels? I am not allowed to edit labels myself.

**Comment by Yajo** [comment]
I'm afraid you still will have to fight a little bit with git to get this ready to migrate. 😅 

1. Get [all commits from previous versions](https://github.com/OCA/partner-contact/commits/8.0/partner_noncommercial). See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 for help.
2. Squash all translation-related commits into one.
3. Squash all migration-related commits into one.

Thanks!

**Review by yajo — CHANGES_REQUESTED** [review]
I'm afraid you still will have to fight a little bit with git to get this ready to migrate. 😅 

1. Get [all commits from previous versions](https://github.com/OCA/partner-contact/commits/8.0/partner_noncommercial). See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 for help.
2. Squash all translation-related commits into one.
3. Squash all migration-related commits into one.

Thanks!

File: partner_noncommercial/README.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Partner Non Commercial
+======================
+
+This module changes menu and view descriptions to make partner management
+suitable for use in non commercial organisations.
```
Comment (pedrobaeza) [inline partner_noncommercial/README.rst:10]: Can you explain a bit more the changes made?

**Comment by hbrunn** [comment]
@NL66278 please follow up on https://github.com/OCA/partner-contact/pull/477#pullrequestreview-70167217, and add a bit to coverage

**Comment by hbrunn** [comment]
@Yajo agree to merge?

File: partner_noncommercial/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV (http://therp.nl).
```
Comment (nikul-serpentcs) [inline partner_noncommercial/__init__.py]: Remove Copyright from all `__init__` file

File: partner_noncommercial/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 Therp BV (http://therp.nl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Partner Non Commercial',
+    'version': '10.0.1.0.0',
+    "author": "Therp BV,Camptocamp,Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline partner_noncommercial/__manifest__.py]: Add Website as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: partner_noncommercial_multi_relation/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Therp BV (http://therp.nl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Partner Non Commercial Multi Relation',
+    'version': '10.0.1.0.0',
+    "author": "Therp BV,Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline partner_noncommercial_multi_relation/__manifest__.py]: same hare

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

**Comment by NL66278** [comment]
@nikul-serpentcs I added the website keys. kept the copyright in all the init files. We have them everywhere in all modules in all python files. You might debate about the use, but would be inconsistent to just change it here.

**Comment by nikul-serpentcs** [comment]
@NL66278 Here we followed OCA terms and in [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py)

**Comment by NL66278** [comment]
@nikul-serpentcs OK, I changed it. Thanx for the links.

**Review by yajo — APPROVED** [review]
Please, could you squash the migration commits, to merge? Thanks!

**Comment by NL66278** [comment]
@Yajo commits have been squashed.

**Comment by yajo** [comment]
Yikes, sorry to ask for this for the 3rd time in the thread, but I think you didn't understand me: Only the *migration* commits.

The process is:

1. Get all commits from the previous branch (you can squash translation commits if you want).
1. Put 1 more commit (squashed) with the migration for this branch.

**Comment by NL66278** [comment]
@Yajo I might have misunderstood your remark on squashing. Anyway I now recreated the commits with the 8.0 history in them. As 8.0 was a migration of 7.0 without the history, that was as far as I could get. Hope it is OK now. Thanks for your patience.

**Comment by yajo** [comment]
Thanks! It seems you still need to fix https://travis-ci.org/OCA/partner-contact/jobs/436795018#L623. Squash the fix with your last commit and tell me, to merge.

**Comment by NL66278** [comment]
@Yajo Fixed!

**Review by bodedra — APPROVED** [review]

**Pull request merged**